source/features/more-dropdown.tsx

```diff
@@ -31,18 +31,45 @@ function createDropdown(): void {
 	);
 }
 
+/* eslint-disable-next-line import/prefer-default-export */
+export function createDropdownItem(label: string, url: string, attributes?: Record<string, string>): Element {
+	return (
+		<li {...attributes}>
+			<a role="menuitem" className="dropdown-item" href={url}>
+				{label}
+			</a>
+		</li>
+	);
+}
+
 async function init(): Promise<void> {
 	await elementReady('.pagehead + *'); // Wait for the tab bar to be loaded
-	if (!select.exists('.reponav-dropdown')) {
-		createDropdown();
-	}
 
 	const reference = getCurrentBranch();
 	const compareUrl = `/${repoUrl}/compare/${reference}`;
 	const commitsUrl = `/${repoUrl}/commits/${reference}`;
+	const dependenciesUrl = `/${repoUrl}/network/dependencies`;
 
-	const menu = select('.reponav-dropdown .dropdown-menu')!;
+	const nav = select('.js-responsive-underlinenav .UnderlineNav-body');
+	if (nav) {
+		// "Repository refresh" layout
+		nav.parentElement!.classList.add('rgh-has-more-dropdown');
+		select('.js-responsive-underlinenav-overflow ul')!.append(
+			<li className="dropdown-divider" role="separator"/>,
+			createDropdownItem('Compare', compareUrl),
+			pageDetect.isEnterprise() ? '' : createDropdownItem('Dependencies', dependenciesUrl),
```

Comment:
can this also use `&&`?

Reply:
No because `.append` will stringify the Boolean, unlike dom-chef